src/vuex/modules/wallet.js

```diff
@@ -0,0 +1,48 @@
+// creates a cosmos addres for the network desired
```

Comment:
move to `src/crypto`? we could move their all those crypto scripts

Reply:
you mean `src/scripts`? I cannot find any `src/crypto`

Reply:
was a proposal for a new location, we can do that later